<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/MainWindow/newBackgrounds.jpg</normaloff>:/MainWindow/newBackgrounds.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 25 10pt &quot;Microsoft JhengHei UI Light&quot;;

}





QPushButton{
border-radius: 10px;
border: transparent;
padding: 0 10px;
background-color: rgb(220, 220, 220);

}

QPushButton::hover{
background-color:rgb(199, 245, 255);
}


QPushButton::hover::pressed{
background-color:rgb(151, 255, 221);
}


QComboBox {
    background: transparent;
    border: 2px solid white;
	border-radius: 3px;
    padding: 3px;
 background-color: rgb(255, 255, 255);
}


QComboBox::on{
	background: 2px solid white; 
}

QLineEdit
{
background: transparent;
border-top: transparent;
border-left: transparent;
border-right: transparent;
border-bottom: 2px solid grey;

}


QMenuBar{
background: transparent;
border-top: transparent;
border-left: transparent;
border-right: transparent;
}

QMenuBar::item:selected{
background: 2px solid white;
}


QMenu{
	background:2px solid white;
}


QMenu::item::selected{
	background:2px solid rgb(151, 255, 221);
}

QTableView
{
background:rgb(255, 255, 255);
border: 2px solid white;
}

</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableView" name="tableView">
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string>Status...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_3">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="3">
          <widget class="QLineEdit" name="searchByUserEdit">
           <property name="placeholderText">
            <string>Search By ID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="findButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="tableComboBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft JhengHei UI Light</family>
             <pointsize>10</pointsize>
             <weight>3</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <item>
            <property name="text">
             <string>Employee</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Patient</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Doctor</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="loadTableButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>load Table</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QLabel" name="displayUserLable">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="userToolButton">
        <property name="styleSheet">
         <string notr="true">border-image: url(C:/CodingWorkSpace/Icons and Images/usercolor.jpg);</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Database Management System</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServices">
    <property name="title">
     <string>Service</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionAddEmployee"/>
     <addaction name="separator"/>
     <addaction name="actionAddPatient"/>
     <addaction name="separator"/>
     <addaction name="actionAddDoctor"/>
    </widget>
    <widget class="QMenu" name="menuFnd">
     <property name="title">
      <string>Find</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionEmployeeFind"/>
     <addaction name="separator"/>
     <addaction name="actionPatientFind"/>
     <addaction name="separator"/>
     <addaction name="actionFindDoctor"/>
    </widget>
    <widget class="QMenu" name="menuAdministration">
     <property name="title">
      <string>Administration</string>
     </property>
     <widget class="QMenu" name="menuRemove">
      <property name="title">
       <string>Remove</string>
      </property>
      <addaction name="separator"/>
      <addaction name="actionEmployeeRemove"/>
      <addaction name="separator"/>
      <addaction name="actionPatientRemove"/>
      <addaction name="separator"/>
      <addaction name="actionRemoveDoctor"/>
     </widget>
     <widget class="QMenu" name="menuUpdate">
      <property name="title">
       <string>Update</string>
      </property>
      <addaction name="separator"/>
      <addaction name="actionEmployeeUpdate"/>
      <addaction name="separator"/>
      <addaction name="actionPatientUpdate"/>
      <addaction name="separator"/>
     </widget>
     <addaction name="separator"/>
     <addaction name="menuRemove"/>
     <addaction name="separator"/>
     <addaction name="menuUpdate"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="menuFnd"/>
    <addaction name="separator"/>
    <addaction name="menuAdministration"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout_Us"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuLogin_Register">
    <property name="title">
     <string>Login/Register</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionRegister"/>
   </widget>
   <addaction name="menuServices"/>
   <addaction name="menuLogin_Register"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddEmployee">
   <property name="text">
    <string>Employee</string>
   </property>
  </action>
  <action name="actionAddPatient">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionEmployeeFind">
   <property name="text">
    <string>Employee</string>
   </property>
  </action>
  <action name="actionPatientFind">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionEmployeeRemove">
   <property name="text">
    <string>Employee</string>
   </property>
  </action>
  <action name="actionPatientRemove">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionEmployeeUpdate">
   <property name="text">
    <string>Employee</string>
   </property>
  </action>
  <action name="actionPatientUpdate">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionAddDoctor">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
  <action name="actionFindDoctor">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
  <action name="actionRemoveDoctor">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
  <action name="actionUpdateDoctor">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
